addons/addon-base-raas-ui/packages/base-raas-ui/src/models/forms/AddUpdateAwsAccountForm.js

```diff
@@ -33,7 +33,7 @@ const addBaseAwsAccountFormFields = {
   },
 };
 
-const addAwsAccountAppStreamFormFields = {
+const addUpdateAwsAccountAppStreamFormFields = {
```

Comment:
Could we enter default values in this form?

Reply:
Hmm, we could do that, but then users won't be able to see the description of the field in the placeholder value. I'm leaning towards not having a default value so they can be informed of their decision and select the values that best suit their need. Thoughts? 
![Screen Shot 2021-07-28 at 5 04 05 PM](https://user-images.githubusercontent.com/3661906/127395672-66911c15-bd25-4f9c-b216-3c0b8f981a95.png)